chaos_genius/alerts/anomaly_alerts.py

```diff
@@ -1022,13 +1028,24 @@ def _send_email_alert(
         )
 
         # attach CSV of anomaly data
+        filename_date = individual_data.date.strftime(
+            ALERT_DATE_CSV_FILENAME_FORMAT_INDIVIDUAL
+        )
         files = [
             {
-                "fname": "data.csv",
-                "fdata": _make_anomaly_data_csv(
-                    individual_data.all_points,
-                    include_subdims=individual_data.include_subdims,
+                "fname": (
+                    f"chaosgenius_alert_{individual_data.kpi_name}"
```

Comment:
We need to make the kpi name path friendly. Right now, the kpi name can include any character. We should convert the KPI name to only alphanumeric character (replacing space with underscore[optional])

Reply:
Agreed. I think spaces are fine, but Windows has issues with colon, backslash, etc. I'll make a util function for this and use it here.